src/util/FactorUtil.js

```diff
@@ -106,6 +106,12 @@ function (Okta, TimeUtil) {
       iconClassName: 'mfa-okta-security-question',
       sortOrder: 12
     },
+    'PASSWORD': {
+      label: 'factor.password',
+      description: '',
+      iconClassName: 'mfa-okta-password',
+      sortOrder: 13
```

Comment:
I feel this should be higher sort order... probably 11 (so higher than email and question)

Reply:
But it will not show up in enrollment right now.. Should we change it once we support enroll?

Reply:
ah right... good point. It doesn't matter for now then.